### PR TITLE
Added get by ride skeleton

### PR DESCRIPTION
**Pull request by xuale**

### CODE REVIEW

File: src/components/Card/index.js [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
-      </h3>
-    </Card>
-  );
-};
+class GenericCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.openModal = this.openModal.bind(this);
+  }
+  openModal() {
```
Comment (xuale) [inline src/components/Card/index.js:8]: Passing down getByID() and the id as props; putting functionality inside card. I prefer this to the alternative of rebinding functions inside the container, i.e. when we map trips in dashboard.

File: src/containers/dashboard.js [hunk 2]
```diff
@@ -1,39 +1,73 @@
...
+    this.getRideByID = this.getRideByID.bind(this);
+    this.closeModal = this.closeModal.bind(this);
   }
+
   componentDidMount() {
     this.getRides();
   }
 
-  getRides() {
+  async getRides() {
```
Comment (xuale) [inline src/containers/dashboard.js:23]: will need to be async eventually

File: src/components/Modal/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  render() {
+    const { info } = this.props;
+    const style = ["modal"];
+    if (this.props.scrollbox) {
+      style.push("scrollbox");
+    }
+    return (
+      <div className={style.join(" ")}>
+        <Button className="close-button" onClick={this.closeModal} />
+        <Card>{info}</Card>
```
Comment (xuale) [inline src/components/Modal/index.js]: forgot to add TODO; will have to format the modal differently

File: src/components/Modal/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    this.props.closeModal();
+  }
+  render() {
+    const { info } = this.props;
+    const style = ["modal"];
+    if (this.props.scrollbox) {
+      style.push("scrollbox");
+    }
+    return (
+      <div className={style.join(" ")}>
```
Comment (rishub) [inline src/components/Modal/index.js]: maybe do this instead:
```
const { info, scrollbox } = this.props

<div className={`modal ${scrollbox ? "scrollbox" : ""}`}>
```
personal preference but either way works

Comment (xuale) [reply src/components/Modal/index.js]: I'll try this. Definitely is less verbose

**Pull request merged**